src/test/java/org/jabref/migrations/SpecialFieldsToSeparateFieldsTest.java

```diff
@@ -87,7 +87,7 @@ private static Stream<Arguments> provideKeywordFieldPairs() {
                         SpecialField.PRIORITY, "prio1", new BibEntry().withField(SpecialField.PRIORITY, "prio1")
                 ),
                 Arguments.of(
-                        SpecialField.QUALITY, "qualityAssured", new BibEntry().withField(SpecialField.QUALITY, "qualityAssured")
+                        SpecialField.QUALITY, "quality1", new BibEntry().withField(SpecialField.QUALITY, "quality1")
```

Comment:
Altering test cases should be done with care. Why is the old field `qualityAssuered` removed? What about backwards compatibility?